From: Joe Rabinoff <jrabinoff6@math.gatech.edu>
Date: Mon, 18 Jun 2018 10:38:46 -0400
Subject: Change "const" to "var"

These variables are in fact used from other modules, so gjs complains about them
being const.

(cherry picked from commit 8237a1f6e002266696eb93e7cf15d3be7191585f)

Applied-upstream: 3.28.3, commit:4ff7e84c5192722d10035cff7b84d524386d6df5
---
 js/ui/windowManager.js      | 2 +-
 js/ui/workspaceThumbnail.js | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/js/ui/windowManager.js b/js/ui/windowManager.js
index d450f35..17576a0 100644
--- a/js/ui/windowManager.js
+++ b/js/ui/windowManager.js
@@ -24,7 +24,7 @@ const EdgeDragAction = imports.ui.edgeDragAction;
 const CloseDialog = imports.ui.closeDialog;
 const SwitchMonitor = imports.ui.switchMonitor;
 
-const SHELL_KEYBINDINGS_SCHEMA = 'org.gnome.shell.keybindings';
+var SHELL_KEYBINDINGS_SCHEMA = 'org.gnome.shell.keybindings';
 var MINIMIZE_WINDOW_ANIMATION_TIME = 0.2;
 var SHOW_WINDOW_ANIMATION_TIME = 0.15;
 var DIALOG_SHOW_WINDOW_ANIMATION_TIME = 0.1;
diff --git a/js/ui/workspaceThumbnail.js b/js/ui/workspaceThumbnail.js
index c1b4bdd..e11f32a 100644
--- a/js/ui/workspaceThumbnail.js
+++ b/js/ui/workspaceThumbnail.js
@@ -31,7 +31,7 @@ var WORKSPACE_CUT_SIZE = 10;
 
 var WORKSPACE_KEEP_ALIVE_TIME = 100;
 
-const OVERRIDE_SCHEMA = 'org.gnome.shell.overrides';
+var OVERRIDE_SCHEMA = 'org.gnome.shell.overrides';
 
 /* A layout manager that requests size only for primary_actor, but then allocates
    all using a fixed layout */
@@ -241,7 +241,7 @@ var WindowClone = new Lang.Class({
 Signals.addSignalMethods(WindowClone.prototype);
 
 
-const ThumbnailState = {
+var ThumbnailState = {
     NEW   :         0,
     ANIMATING_IN :  1,
     NORMAL:         2,
